[CI] InternalExtendedStatsTests.testReduceRandom fails assertion

https://github.com/elastic/elasticsearch/issues/24931 popped up again on current master:

reproduce with: 

```
./gradlew :server:test -Dtests.seed=97B9AEB9875197B5 -Dtests.class=org.elasticsearch.search.aggregations.metrics.InternalExtendedStatsTests -Dtests.method="testReduceRandom" -Dtests.security.manager=true -Dtests.locale=de-CH -Dtests.timezone=Europe/Sofia
```

```
FAILURE 0.03s J2 | InternalExtendedStatsTests.testReduceRandom <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: expected:<321.2197163576493> but was:<321.2197163657984>
   > 	at __randomizedtesting.SeedInfo.seed([97B9AEB9875197B5:578402A8A10C0EE3]:0)
   > 	at org.elasticsearch.search.aggregations.metrics.InternalExtendedStatsTests.assertReduced(InternalExtendedStatsTests.java:87)
   > 	at org.elasticsearch.search.aggregations.metrics.InternalExtendedStatsTests.assertReduced(InternalExtendedStatsTests.java:37)
   > 	at org.elasticsearch.test.InternalAggregationTestCase.testReduceRandom(InternalAggregationTestCase.java:262)
```

--- a/server/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStatsTests.java
+++ b/server/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStatsTests.java
@@ -84,7 +84,7 @@ public class InternalExtendedStatsTests extends InternalAggregationTestCase<Inte
         assertEquals(expectedCount, reduced.getCount());
         // The order in which you add double values in java can give different results. The difference can
         // be larger for large sum values, so we make the delta in the assertion depend on the values magnitude
-        assertEquals(expectedSum, reduced.getSum(), Math.abs(expectedSum) * 1e-11);
+        assertEquals(expectedSum, reduced.getSum(), Math.abs(expectedSum) * 1e-10);
         assertEquals(expectedMin, reduced.getMin(), 0d);
         assertEquals(expectedMax, reduced.getMax(), 0d);
         // summing squared values, see reason for delta above

